src/wallet/rpc/transactions.cpp

```diff
@@ -303,6 +303,16 @@ static void MaybePushAddress(UniValue & entry, const CTxDestination &dest)
     }
 }
 
+//! Fetch parent descriptors of this scriptPubKey.
+static void PushParentDescriptors(const CWallet& wallet, const CScript& script_pubkey, UniValue& entry)
```

Comment:
In 2a7f812df2ac5c809b4690f9d956564108f705f8 "rpc: output wallet descriptors for received entries in listsinceblock"

To make it easier to read the diffs, perhaps move this to it's final location when you introduce here rather than moving it again in the next commit.

Reply:
Done.